src/main/java/com/theelixrlabs/healthcare/response/SuccessResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.theelixrlabs.healthcare.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Represents a generic success response structure for API responses.
+ *
+ * @param <User> the type of data included in the response.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SuccessResponse<User> {
```

Comment:
@Data lombok annotation includes @NoArgsConstructor and @AllArgsConstructor. You don't need to add it separately. Please check in all places